package.json

```diff
@@ -54,6 +54,7 @@
     "@lhci/cli": "^0.7.2",
     "@mdn/browser-compat-data": "5.2.61",
     "@rollup/plugin-node-resolve": "^8.4.0",
+    "adm-zip": "^0.5.10",
```

Comment:
Please pin the dependency (this one and the related @types package), so remove the `^`. This avoids package manager/environment inconsistencies.